nginx.conf

```diff
@@ -0,0 +1,63 @@
+# auto detects a good number of processes to run
```

Comment:
I am unsure why we have a nginx.conf here

Is it to copy it later on a server?